core/include/detray/propagator/rk_stepper.hpp

```diff
@@ -79,16 +81,12 @@ class rk_stepper final
             vector3 k2{0.f, 0.f, 0.f};
             vector3 k3{0.f, 0.f, 0.f};
             vector3 k4{0.f, 0.f, 0.f};
-            array_t<scalar, 4> k_qop{0.f, 0.f, 0.f, 0.f};
+            array_t<scalar_type, 4> k_qop{0.f, 0.f, 0.f, 0.f};
         } _step_data;
 
         /// Magnetic field view
         const magnetic_field_t _magnetic_field;
 
-        /// Set the local error tolerenace
-        DETRAY_HOST_DEVICE
-        inline void set_tolerance(scalar tol) { _tolerance = tol; };
```

Comment:
Is this necessary?

Reply:
I think, if the navigator consolidates its config, so should the stepper. It would be a bit weird to break that pattern. What was previously set on the state(s) directly, now goes into a small config struct at the same instance the states are created. The parameters were not moved to a different place in the code for all I could see (?). I could also not find that this this particular function is used in traccc?